llvm/include/llvm/IR/DebugInfo.h

```diff
@@ -263,7 +265,8 @@ struct VarRecord {
 /// TODO: Backing storage shouldn't be limited to allocas only. Some local
 /// variables have their storage allocated by the calling function (addresses
 /// passed in with sret & byval parameters).
-using StorageToVarsMap = DenseMap<const AllocaInst *, SmallSet<VarRecord, 2>>;
+using StorageToVarsMap =
+    DenseMap<const AllocaInst *, SmallSetVector<VarRecord, 2>>;
```

Comment:
Can you add a comment explaining the choice of `SmallSetVector` and nondeterminism?  (Just so no one feels tempted to change it back.)

Reply:
I'm not sure that is necessary. SmallSetVector is a type whose main use is to prevent non-deterministic output.